LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2021, Zebedee Nicholls, Daniel Huppmann
```

Comment:
```suggestion
Copyright (c) 2021, Contributors
```

Could seem more inviting to newcomers?

Reply:
Can we add something like 'BSD 3-Clause license' at the top so people know what they're looking at straight away?

Reply:
This is copied straight from the [GitHub license templates](https://github.com/licenses/license-templates/blob/master/templates/bsd3.txt), and the summary and name will be shown as a summary header, same as on [scmdata](https://github.com/openscm/scmdata/blob/master/LICENSE).

Not sure if the automated recognition of the license would still work if we deviate from the standard license text.

Reply:
Ok cool let's leave it then